### ISSUE TITLE
does not find modules with extensions even with resolve/extensions

### ISSUE THREAD

**Issue by graingert** [benmosher/eslint-plugin-import#375]
https://github.com/graingert/eslint-webpack-resolver-fail
```
git clone https://github.com/graingert/eslint-webpack-resolver-fail.git
npm install
/node_modules/.bin/eslint src
/home/xxx/yyy/eslint-webpack-resolver-fail/src/summary/index.es2015.js
  1:8  error  Unable to resolve path to module './foo'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)

```

**Comment by benmosher** [comment]
Bizarre. Everything looks good. Thanks for the complete repro! I will take a closer look when I can. 

One thought: 1.9.0 is broken for the Webpack installer. It shouldn't have installed but you might try explicitly installing 1.8.1. I'm not sure if the ^1.8.1 version string will install 1.9.0 since it's set to the "next" npm tag.

**Comment by graingert** [comment]
@benmosher I'm definitely not using 1.9.0 see [the version in my recently added shrinkwrap](https://github.com/graingert/eslint-webpack-resolver-fail/blob/527549a12b80d343a4f8488a73ff77fe3418a454/npm-shrinkwrap.json#L503-L507).

**Comment by benmosher** [comment]
Fixed your config in a PR to your example repo. 

*TL;DR*: it's `import/resolver: ...`, not `import: { resolver: ... }`

**Comment by graingert** [comment]
@benmosher thanks!
